autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/SecureStoreBackedAutofillStore.kt

```diff
@@ -168,13 +172,21 @@ class SecureStoreBackedAutofillStore(
             }
     }
 
+    override suspend fun getAllCredentialsModifiedSince(modifiedSince: Long): List<LoginCredentials> {
+        return secureStorage.websiteLoginDetailsWithCredentialsModifiedSince(modifiedSince)
+            .map {
+                it.toLoginCredentials()
+            }
+    }
+
     override suspend fun getCredentialCount(): Flow<Int> {
         return secureStorage.websiteLoginDetailsWithCredentials().map { it.size }
     }
 
     override suspend fun deleteCredentials(id: Long): LoginCredentials? {
         val existingCredential = secureStorage.getWebsiteLoginDetailsWithCredentials(id)
         secureStorage.deleteWebsiteLoginDetailsWithCredentials(id)
+        credentialsSyncMetadata.onEntityRemoved(id)
```

Comment:
Probably related to https://app.asana.com/0/0/1205110657412668/f

Reply:
all good here (when user deletes, we update delete timestamp in metadata db)